src/redux/actionCreators.js

```diff
@@ -0,0 +1,34 @@
+import ACTION_TYPES from "./actionTypes";
+
+const addPostCreator = () => {
```

Comment:
There's a shorter version for arrow functions that return an object.
https://mariusschulz.com/blog/returning-object-literals-from-arrow-functions-in-javascript

Reply:
done